operations/security_and_authentication/Troubleshoot_Kyverno_Configuration_manually.md

```diff
@@ -0,0 +1,25 @@
+# How to troubleshoot Kyverno configuration manually
+
+## Check if Kyverno pods are up and running
+
+Run the script with -p option.
+
+```bash
+./k8s_kyverno_pods_running.sh -p
+```
```

Comment:
Please make the same change as the other PR - fully qualified path to the script.

Reply:
Stale comment, based on an offline discussion this can merge.

Reply:
Fixed.